cmd/sos_sync.go

```diff
@@ -0,0 +1,518 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"github.com/minio/minio-go/v6"
+	"github.com/vbauerster/mpb/v4"
+	"github.com/vbauerster/mpb/v4/decor"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+//region Settings
```

Comment:
Can you please remove extraneous comments? We try to use comments only when it's mandated by the linter or when it makes sense for the reader, to give context as why the code is implemented a specific way.